File: Java/Ciclofor.java
import java.util.Scanner; // Importar Scanner para leer la entrada del usuario

public class Ciclofor {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in); // Crear un objeto Scanner

        // Solicitar al usuario que ingrese el primer valor
        System.out.print("Ingrese el primer valor: ");
        int A = scanner.nextInt(); // Leer el primer número como entero

        // Solicitar al usuario que ingrese el segundo valor
        System.out.print("Ingrese el segundo valor: ");
        int C = scanner.nextInt(); // Leer el segundo número como entero

        // Calcular la potencia de A elevado a C
        double valor = Math.pow(A, C); // Se usa Math.pow() para calcular la potencia

        // Mostrar el resultado
        System.out.println("La potencia de " + A + " sobre " + C + " es: " + valor);

        scanner.close(); // Cerrar el scanner
    }
}

File: C mas/Ciclofor.cpp
#include <iostream>
#include <cmath> // Importar librería para usar la función pow()

using namespace std;

int main()
{
    // Declarar variables
    int A, C;

    // Solicitar al usuario que ingrese el primer valor
    cout << "Ingrese el primer valor: ";
    cin >> A; // Leer el primer número como entero

    // Solicitar al usuario que ingrese el segundo valor
    cout << "Ingrese el segundo valor: ";
    cin >> C; // Leer el segundo número como entero

    // Calcular la potencia de A elevado a C
    double valor = pow(A, C); // Se usa pow() de la librería cmath

    // Mostrar el resultado
    cout << "La potencia de " << A << " sobre " << C << " es: " << valor << endl;

    return 0;
}
